<launch>

  <node name="flask_ros_interface" pkg="flask_robot_ros" type="flask_ros_interface.py" respawn="true" output="screen">
      <!-- sub -->
      <remap from="map" to="/map"/>
      <remap from="object_markers" to="/object_markers"/>
      <!-- pub -->
      <remap from="move_base_simple/goal" to="/move_base_simple/goal"/>

      <param name="/ros_interface/map_frame" type="string" value="map"/>
      <param name="/ros_interface/robot_frame" type="string" value="base_link"/>
  </node>

</launch>